mmf/utils/dataset.py

```diff
@@ -20,8 +20,8 @@ def build_bbox_tensors(infos, max_length):
 
     for idx, info in enumerate(infos):
         bbox = info["bounding_box"]
-        x = bbox["top_left_x"]
```

Comment:
I think this is due to older textvqa imdbs. Let's keep the backwards compatibility by changing this and the line below to:
```py
x = bbox.get("top_left_x", bbox["topLeftX"])
y = bbox.get("top_left_y", bbox["topLeftY"])
```